### PR TITLE
feat:implementing the service section

### PR DESCRIPTION
**Pull request by brendahuwitonze**
Here are what this PR does:

- [x] Added the service section which contains
- [x] Positioning The text
- [x] position the vector image with text on it.

### CODE REVIEW

File: src/output.css [hunk 1]
```diff
@@ -574,6 +609,45 @@ video {
   line-height: 2rem;
 }
 
+<<<<<<< HEAD
```
Comment (Kevin-gram) [inline src/output.css:598]: you have conflicts here

File: src/output.css [hunk 2]
```diff
@@ -560,11 +560,46 @@ video {
...
+
+.justify-center{
+  justify-content: center;
+}
+
+>>>>>>> d8a8e79df8e093f5a4b978477fec725f3c93f438
 .bg-green-600{
   --tw-bg-opacity: 1;
   background-color: rgb(22 163 74 / var(--tw-bg-opacity));
 }
```
Comment (Kevin-gram) [inline src/output.css]: there are conflicts here too

File: src/sections/about.html [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+      brand building, identity sharpening, or communication strategy, we bring a
+      fresh perspective to established structures, develop tailor-made
+      communication solutions, and ensure that messages resonate. With over a
+      decade of experience, we have honed our skills in giving brands their
+      unmistakable identity and making them visible in the long term.
+    </p>
+    <div class="flex justify-center mt-16 items-center sticky top-0">
+      <img src="/img/Vector-12.webp">
+  
+    </div>
```
Comment (Kevin-gram) [inline src/sections/about.html]: Consider removing this space  here

**Review by DevRamona — CHANGES_REQUESTED** [review]
Remove the space here.

**Review by Kevin-gram — CHANGES_REQUESTED** [review]
check the comments and make some slight changes

File: ToDo/index.html [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+</head>
+<body> 
+    <!-- <button id="changeColorButton">Change Background Color</button> -->
+    <!-- <button id="toggleVisibilityButton">Toggle Visibility</button>
+    <button id="increaseFontSizeButton">Increase Font Size</button> -->
+    <!-- <p id="targetElement">This is the target element.</p> -->
+    
+<!-- <button id="changeTextButton">Change Text</button>
+<button id="removeTextButton">Remove Text</button>
+<p id="targetElement">Original Text</p>  -->
```
Comment (Gasorekibo) [inline ToDo/index.html:14]: Please consider removing extra comments

**Review by Gasorekibo — CHANGES_REQUESTED** [review]
May you remove those commented codes for good practices?

File: src/sections/service.html [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+    <section class="px-[2rem] md:pt-36 md:grid md:grid-cols-2 md:grid-rows-1 md:justify-end md:gap-48 md:pl-24 md:pr-[10.5rem] md:min-h-[48.75rem] md:min-w-[71.973rem] md:pb-72 pb-72">
+        <div class="md:col-span-1 md:row-span-1 space-y-16 font-body lg:text-5xl  md:text-4xl text-2xl font-medium">
+            <p>Brand strategy</p>
+            <p>Corporate Identity & <br> Website development</p>
+            <p>SEO & copywriting</p>
+            <p>Media Relations</p>
+            <p>Digital marketing & <br> content production</p>
+            <p>Influencer relations</p>
+        </div>
+        <div class="  md:col-span-1  md:row-span-1 bg-[url('../../img/Vector-5.webp')] md:w-[29.285rem] h-[100%] md:bg-contain bg-contain bg-no-repeat md:mt-0 mt-16 ">
```
Comment (Gasorekibo) [inline src/sections/service.html:35]: There is an extra white space between classes. May you remove it? Please consider not using arbitrary(%) values as requested in the morning stand-up.

**Review by Gasorekibo — CHANGES_REQUESTED** [review]

**Pull request merged**